apps/vaporgui/MainForm.cpp

```diff
@@ -2253,7 +2253,6 @@ void MainForm::launchPythonVariables()
 {
     if (!_pythonVariables) {
         _pythonVariables = new PythonVariables(this);
-        connect(_pythonVariables, &QDialog::finished, this, &MainForm::_pythonClosed);
```

Comment:
It seems `MainForm::_pythonClosed` is now dead code so it may make sense to delete it?

Reply:
Makes sense.  Fixed.